cmd/influx/main.go

```diff
@@ -27,6 +29,12 @@ import (
 
 const maxTCPConnections = 10
 
+var (
+	version = "dev"
+	commit  = "none"
+	date    = time.Now().UTC().Format(time.RFC3339)
```

Comment:
Fly-by comment: this isn't the build date (as noted in the PR description); it's the time that the `influx` process was started.

Reply:
you're right. I borrowed this from the influxd binary here:

https://github.com/influxdata/influxdb/blob/master/cmd/influxd/main.go#L27

just noticed I dont' have the `fmt.Sprintf`, but that shouldn't matter yah?

It looks like the date isn't provided as an LD flag either, so that definitely can change 🤔. What do you think is the best way forward here?

Reply:
Agreed the `fmt.Sprint` isn't necessary; `time.Format` already returns a string.

I think setting the date at link time is the correct approach.

Reply:
Looks like we do set the date as a linker flag in goreleaser after all:
https://github.com/influxdata/influxdb/blob/d21fce5f6d026904a6048704c6761384ce2736d9/.goreleaser.yml#L19
https://github.com/influxdata/influxdb/blob/d21fce5f6d026904a6048704c6761384ce2736d9/.goreleaser-nightly.yml#L19

Any tooling we have to inspect the resulting date would know that `version == "dev"` indicates the date would be meaningless.

Nothing to change here after all.